### ISSUE TITLE
Buy limit on config.yml

### ISSUE THREAD

**Issue by IrineuMC** [Leonidius20/TradingInterface#10]
Add an option, in config.yml, to define the limit number that the person has to buy the item per purchase screen! Because it is very bad when selecting, with the standard limit, when the person only wants to buy 2, for example, and can only select 0 or 6.

Example in config.yml:
buy-limit: 128

**Comment by Leonidius20** [comment]
You can actually select anything between 0 and 6, but the slider is a little buggy and it sometimes jumps straight to the max amount when you drag it. Drag it gently back to 2 or whatever number you want to select and it's gonna work

**Comment by Leonidius20** [comment]
Or did you mean that it's hard to select a certain number when the max number is really big?

**Comment by IrineuMC** [comment]
Yes, it's hard to select a certain number when the max number is really big!
